client/src/components/table-components/Dealer.js

```diff
@@ -1,7 +1,110 @@
+import "../../App.css";
+import React, { useState, useEffect } from "react";
+import convertSecondstoTime from "../../utils/timeConversor.js";
```

Comment:
Please, do a research on how to use imports with absolute paths in react. That avoids '../..' at the imports and make things easier to find when the project grows.